### ISSUE TITLE
Filtering Underlying Data

### ISSUE THREAD

**Issue by Mrkbingham** [tableau/embedding-api-v3-guide#7]
It looks like in the [Querying Data section](https://github.com/tableau/embedding-api-v3-guide#querying-data) Summary Data will be available to be retrieved on a per column basis (very happy about that!)

Will this also be available for Underlying data?  It would likely help resolve [this use case](https://github.com/tableau/embedding-api-v3-guide/issues/6)

**Comment by Kovner** [comment]
Yes actually and thank you for bringing it to our attention that that's not in the migration guide.
In general, any new functionality we've added to the Extensions API will make it to the Embedding API where appropriate. As you can see in the [Extensions API reference](https://tableau.github.io/extensions-api/docs/interfaces/getunderlyingdataoptions.html), we have a columnsToIncludeById option that you can pass in to getUnderlyingDataAsync

**Comment by Mrkbingham** [comment]
> In general, any new functionality we've added to the Extensions API will make it to the Embedding API where appropriate

Music to my ears 👏
